tempesta_fw/http.h

```diff
@@ -223,7 +225,7 @@ typedef struct {
 #define TFW_HTTP_CHUNKED		0x000004
 
 /* Request flags */
-#define TFW_HTTP_STICKY_SET		0x000100	/* Need 'Set-Cookie` */
+#define TFW_HTTP_HAS_STICKY_COOKIE	0x000100
```

Comment:
Perhaps, just `TFW_HTTP_HAS_STICKY` would be better. Additional `_COOKIE` doesn't add anything to the understanding of this constant.